### PR TITLE
fix(tooltip): fixed tooltip colormode type issue

### PR DESCRIPTION
**Pull request by yiyang-qantashotels**
## Description

💁‍

## Related issues

💪

## Looks like this

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan), [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/Tooltip/Tooltip.jsx [hunk 1]
```diff
@@ -31,7 +36,7 @@ const getColor = (colorMode, theme) => {
 const TooltipWrapper = styled(Box)`
   position: relative;
   display: inline-block;
-  white-space: nowrap;
+  ${whiteSpace}
```
Comment (ajlu) [inline src/components/Tooltip/Tooltip.jsx:39]: Can this be moved down next to `${space}`?

File: src/components/Tooltip/Tooltip.jsx [hunk 2]
```diff
@@ -128,6 +133,7 @@ Tooltip.defaultProps = {
   pointerDirection: 'top',
   colorMode: 'light',
   fixedCloseButton: false,
+  whiteSpace: 'nowrap',
```
Comment (ajlu) [inline src/components/Tooltip/Tooltip.jsx:136]: Add to type def as well?

**Review by ajlu — APPROVED** [review]

**Review by philipwindeyer — APPROVED** [review]
🚀

**Pull request merged**